### PR TITLE
Update SV_WC_Payment_Gateway_Payment_Tokens_Handler::update_tokens()

### PR DESCRIPTION
**Pull request by ChaseWiseman**
# Summary

Migrates token updating to core handling.

## Details

Note: left transient/cache handling alone until we finalize the `::get_tokens()` approach.

### Setup

- Update the NETBilling gateway's composer version to `dev-ch24096/update-tokens`
- Activate and configure the gateway

### Steps

1. Place an order and save your credit card at checkout
    - [ ] The token is added to the database with the correct properties and meta

## Before merge

- [x] I have confirmed these changes in each supported minor WooCommerce version

### CODE REVIEW

File: tests/integration/payment-gateway/SV_WC_Payment_Gateway_Payment_Tokens_Handler_Test.php [hunk 1]
```diff
@@ -26,6 +26,27 @@ protected function _after() {
 	}
 
 
+	/**
+	 * @see Framework\SV_WC_Payment_Gateway_Payment_Tokens_Handler::update_tokens()
+	 */
+	public function test_update_tokens() {
```
Comment (unfulvio) [inline tests/integration/payment-gateway/SV_WC_Payment_Gateway_Payment_Tokens_Handler_Test.php]: shouldn't we add tests about how data changes when it's updated? currently this only tests that data is retrieved, but there's no insight about its contents

File: woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-tokens-handler.php [hunk 2]
```diff
@@ -613,8 +613,28 @@ public function update_tokens( $user_id, $tokens, $environment_id = null ) {
...
+			// TODO: set the environment ID {CW 2019-12-17}
+
+			$token->save();
+
+			if ( $core_token = $token->get_woocommerce_payment_token() ) {
+				$updated_tokens[] = $core_token->get_token();
+			}
+		}
+
+		return ! empty( $updated_tokens );
```
Comment (unfulvio) [inline woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-tokens-handler.php]: how about we return true/false based on the combined results of `\WC_Data::save()` ? that should return theoretically true/false if save was successful

File: tests/integration/payment-gateway/SV_WC_Payment_Gateway_Payment_Tokens_Handler_Test.php (hunk 1 above)
Comment (ChaseWiseman) [reply tests/integration/payment-gateway/SV_WC_Payment_Gateway_Payment_Tokens_Handler_Test.php]: I figured any tests for ensuring different properties/meta are set should be testing the `::save()` method specifically. In the update tokens context, we're just testing that it was saved

File: woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-tokens-handler.php (hunk 2 above)
Comment (ChaseWiseman) [reply woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-tokens-handler.php]: Added c3d078b7d67c7dddd81a7dad21deadf3fe313e35 to tweak the result a bit.

We'll actually probably end up removing this return and phasing out the use of update_tokens as we update the single token handler methods, as it doesn't hold much value either way. There's only one spot where its results are used, via `->add_token()`, so this return mimics that result, i.e. all of the tokens were or were not saved.

**Pull request merged**